internal/zinc-scripted/src/test/scala/sbt/internal/inc/IncHandler.scala

```diff
@@ -118,9 +118,12 @@ class IncHandler(directory: Path, cacheDir: Path, scriptedLog: ManagedLogger, co
 
   def initBuildStructure(): Unit = {
     val build = initBuild
+    val defaultScalaVersion = "2.13.12"
```

Comment:
Would it be better for `defaultScalaVersion` to be stored in `sys.props` instead?

Like https://github.com/sbt/zinc/blob/3e0ea937035c0319faad32899fe5ce842259be2c/internal/compiler-bridge-test/src/test/scala/sbt/internal/inc/CompilingSpecification.scala#L33-L36

Then we can conveniently setup CI to run scripted tests against both 2.12 & 2.13 in a clean manner.

Reply:
@xuwei-k Since you did lots of work on the CI & Testing side, would you mind to give a quick look on this PR?